docs/custom_skip_members.py

```
@@ -1,30 +1,15 @@
+def autoapi_skip_members(app, what, name, obj, skip, options):
+    if name in skip_dict.get(what, []):
+        skip = True
+
+    if name in keep_dict.get(what, []):
+        skip = False
+
+    return skip
+
+
+def setup(app):
+   app.connect("autoapi-skip-member", autoapi_skip_members)
```

Pylint (3.3.7)

[warning] Bad indentation. Found 3 spaces, expected 4

(W0311)